db/seeds.rb

```diff
@@ -6,39 +6,10 @@
-admin_role = Role.create(name: 'admin', permission_attributes:
-                        { can_lock_user: true, can_edit_role: true, can_edit_permission: true,
-                          can_read_rating: true, can_give_rating: true,
-                          can_set_role: true, can_edit_movie: true, can_read_movie: true,
-                          can_edit_person: true, can_read_person: true,
-                          can_read_entities: true, can_edit_entities: true, can_read_user: true })
-redactor_role = Role.create(name: 'redactor', permission_attributes:
-                        { can_lock_user: false, can_edit_role: false, can_edit_permission: false,
-                          can_read_rating: true, can_give_rating: true,
-                          can_set_role: false, can_edit_movie: true, can_read_movie: true,
-                          can_edit_person: true, can_read_person: true,
-                          can_read_entities: true, can_edit_entities: true, can_read_user: true })
-reviewer_role = Role.create(name: 'reviewer', permission_attributes:
-                        { can_lock_user: false, can_edit_role: false, can_edit_permission: false,
-                          can_read_rating: true, can_give_rating: true,
-                          can_set_role: false, can_edit_movie: false, can_read_movie: true,
-                          can_edit_person: false, can_read_person: true,
-                          can_read_entities: false, can_edit_entities: false, can_read_user: true })
-
-User.create(email: 'admin@example.com', password: '123456', password_confirmation: '123456',
-            name: Faker::Name.first_name, surname: Faker::Name.last_name,
-            username: Faker::Internet.unique.username(specifier: 8),
-            birthday: Faker::Date.birthday(min_age: 18, max_age: 65),
-            role_id: admin_role.id, confirmed_at: Time.zone.now)
-
-User.create(email: 'redactor@example.com', password: '123456', password_confirmation: '123456',
-            name: Faker::Name.first_name, surname: Faker::Name.last_name,
-            username: Faker::Internet.unique.username(specifier: 8),
-            birthday: Faker::Date.birthday(min_age: 18, max_age: 65),
-            role_id: redactor_role.id, confirmed_at: Time.zone.now)
-User.create(email: 'reviewer@example.com', password: '123456', password_confirmation: '123456',
-            name: Faker::Name.first_name, surname: Faker::Name.last_name,
-            username: Faker::Internet.unique.username(specifier: 8),
-            birthday: Faker::Date.birthday(min_age: 18, max_age: 65),
-            role_id: reviewer_role.id, confirmed_at: Time.zone.now)
 FactoryBot.create_list(:rating, 20)
+FactoryBot.create_list(:permission, 1, :admin)
+FactoryBot.create_list(:permission, 1, :redactor)
+FactoryBot.create_list(:permission, 1, :reviewer)
+FactoryBot.create_list(:user, 1, :admin)
+FactoryBot.create_list(:user, 1, :redactor)
+FactoryBot.create_list(:user, 1, :reviewer)
```

Comment:
this is not good for your teammates, how do we find out what mail and password a particular user has, the previous seeds were made so that you can immediately log in as an admin, editor or reviewer, knowing what data they have. Please improve this

Reply:
I agree, fixed